src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -616,6 +616,16 @@ public T execute(final String merchantAccount, final PaymentData paymentData, fi
         }
     }
 
+    private final Map<String, Object> propertiesToMapWithPropertyFiltering(final Iterable<PluginProperty> properties, TenantContext context) {
+        final Map<String, Object> map = PluginProperties.toMap(properties);
+        for (String sensitiveKey : getConfigProperties(context).getSensitivePropertyKeys()) {
```

Comment:
Nit: I'd materialize `getConfigProperties(context).getSensitivePropertyKeys()`. In an indexed `for` loop, it would be evaluated before each block execution.

Reply:
Done in a138ce2